logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -112,7 +113,7 @@ interface ConversationRepository {
 
     suspend fun getConversationList(): Either<StorageFailure, Flow<List<Conversation>>>
     suspend fun observeConversationList(): Flow<List<Conversation>>
-    suspend fun observeConversationListDetails(includeArchived: Boolean): Flow<List<ConversationDetails>>
+    suspend fun observeConversationListDetails(fromArchived: Boolean): Flow<List<ConversationDetails>>
```

Comment:
If we want to use the past participle it would be better to append `xxxConversations` to the variable. Otherwise I'd prefer shorter form `fromArchive` instead. Just personal choice 😄